lib/src/strings.dart

```diff
@@ -279,7 +294,7 @@ final Map<int, String> unprintableCharCodes = {
   8233: '\\P', //  Escaped Unicode paragraph separator (#x2029) character.
 };
 
-/// List of escape characters. In particular, \x32 is not included because it
+/// List of escape characters. In particular, \x22 is not included because it
```

Comment:
Was this an intentional change? - not sure I understand what this is

Reply:
Remove the comment, because I didn't understand it.

`\x32` is supposedly "2"